frameworks/python3/unittestwrapper.py

```diff
@@ -27,9 +30,9 @@ def addSuccess(self, test):
     super(CwTestResult, self).addSuccess(test)
 
   def addFailure(self, test, err):
-    print("<FAILED::>" + formatMessage(err[1]))
+    print("\n<FAILED::>" + formatError(err))
     super(CwTestResult, self).addFailure(test, err)
 
   def addError(self, test, err):
-    print("<ERROR::>Unhandled Exception: " + formatMessage(err[1]))
+    print("\n<ERROR::>" + formatError(err))
```

Comment:
I don't think this file (or any file in frameworks/python3) is used?

https://github.com/Codewars/codewars-runner-cli/blob/dbf943d62c8d6ed451d27ac1817c0f13b1de558c/lib/runners/python.js#L110

No need to revert the change though.

Reply:
Yeah I didn't think so either, fixed it just in case it gets used again 😄